Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -27,6 +30,8 @@ void RegExpConstructor::initialize(Realm& realm)
     // 22.2.5.1 RegExp.prototype, https://tc39.es/ecma262/#sec-regexp.prototype
     define_direct_property(vm.names.prototype, realm.intrinsics().regexp_prototype(), 0);
 
+    u8 attr = Attribute::Writable | Attribute::Configurable;
+    define_native_function(realm, vm.names.escape, escape, 1, attr);
```

Comment:
Why the separate `attr`? If it's to reduce the line length, we have longer lines further down :^)

Reply:
Consistency - it's what we do in every constructor and prototype:
https://github.com/LadybirdBrowser/ladybird/blob/f87041bf3a8c76dce547dbd1f64d32b119f5468c/Libraries/LibJS/Runtime/MapConstructor.cpp#L32-L33

https://github.com/LadybirdBrowser/ladybird/blob/f87041bf3a8c76dce547dbd1f64d32b119f5468c/Libraries/LibJS/Runtime/PromisePrototype.cpp#L30-L33

Reply:
Interesting!

Reply:
tbh we probably should have made that a constant somewhere years ago